package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fecha",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

Comment:
Could you remove this change? I will the version in a separate commit. I may bundle this change with another change.